pkg/reconciler/v1alpha1/eventlistener/eventlistener_test.go

```diff
@@ -48,16 +50,15 @@ func init() {
 			bldr.EventListenerServiceAccount("sa"),
 		),
 		bldr.EventListenerStatus(
-			bldr.EventListenerConfig(
-				generatedResourceName,
-				"",
-			),
+			bldr.EventListenerConfig(generatedResourceName),
 		),
 	)
 }
 
 var (
-	generatedResourceName string
+	generatedResourceName    string
+	ignoreLastTransitionTime = cmpopts.IgnoreTypes(apis.Condition{}.LastTransitionTime.Inner.Time)
```

Comment:
TIL

Reply:
Nice cleanup for sure

Reply:
:) I picked it up from the pipeline reconciler tests!